src/pages/edge/adobe-journey-optimizer-decisioning/tabs/api-reference.md

```diff
@@ -218,6 +265,92 @@ AEPDecisionScope* decisionScope2 = [[AEPDecisionScope alloc] initWithName: @"myS
 }];
 ```
 
+<Variant platform="ios" api="get-propositions-withTimeout" repeat="12"/>
+
+#### Swift
+
+#### Syntax
+
+```swift
+static func getPropositions(for decisionScopes: [DecisionScope], 
+                            timeout: TimeInterval,
+                            _ completion: @escaping ([DecisionScope: OptimizeProposition]?, Error?) -> Void)
+```
+
+* _decisionScopes_ is an array of decision scopes for which propositions are requested.
+* _timeout_ is a duration in milliseconds specifying the maximum time `getProposition` will wait for completion before returning `Error`.
```

Comment:
Is this supposed to be `_timeout_ is a duration in seconds` for Swift?

Reply:
yes, fixed @spoorthipujariadobe